distributed/tests/test_client.py

```diff
@@ -2128,9 +2129,12 @@ async def test_repr_no_memory_limit(c, s, a, b):
 
 @gen_test()
 async def test_repr_localcluster():
-    async with LocalCluster(
-        processes=False, dashboard_address=":0", asynchronous=True
-    ) as cluster, Client(cluster, asynchronous=True) as client:
+    async with (
```

Comment:
Are there any actual changes in this file that I missed or is this just auto-formatting?

Reply:
Just autoformatting.